### PR TITLE
Prevent XXE attacks

### PR DESCRIPTION
**Pull request by ready-research**
Fixes https://github.com/mybatis/generator/issues/803

### CODE REVIEW

File: eclipse/org.mybatis.generator.eclipse.ui/src/org/mybatis/generator/eclipse/ui/content/ConfigVerifyer.java [hunk 1]
```diff
@@ -97,6 +97,9 @@ public boolean isConfigurationFile() {
     private boolean isConfigFile(InputStream inputStream) {
         try {
             SAXParserFactory factory = SAXParserFactory.newInstance();
+            factory.setFeature("http://apache.org/xml/features/disallow-doctype-decl", true);
```
Comment (jeffgbutler) [inline eclipse/org.mybatis.generator.eclipse.ui/src/org/mybatis/generator/eclipse/ui/content/ConfigVerifyer.java:100]: This line breaks the functionality of the plugin. Please remove it.

**Comment by jeffgbutler** [comment]
One of the lines in this PR breaks the Eclipse functionality. I've just added some tests that show the problem. Once you rebase your branch, you can run the Eclipse plugin tests this way:

1. Open a terminal in the .../eclipse/org.mybatis.generator.eclipse.parent directory
2. Run the command `./mvnw -Prelease-composite clean integration-test`

File: eclipse/org.mybatis.generator.eclipse.ui/src/org/mybatis/generator/eclipse/ui/content/ConfigVerifyer.java (hunk 1 above)
Comment (ready-research) [reply eclipse/org.mybatis.generator.eclipse.ui/src/org/mybatis/generator/eclipse/ui/content/ConfigVerifyer.java]: Removed this line.

**Comment by jeffgbutler** [comment]
Thanks for your contribution!

**Comment by ready-research** [comment]
@jeffgbutler Can you please validate the same in huntr by using `Mark as valid` and also `Confirm the fix`. Thanks.
https://huntr.dev/bounties/acd74536-dea0-4507-8c2c-1df3e21a948a/

**Comment by jeffgbutler** [comment]
@ready-research I don't use huntr, but perhaps you can link this PR to whatever entry is in that system.

**Pull request merged**